src/Symfony/Component/DependencyInjection/Tests/ContainerTest.php

```diff
@@ -136,13 +136,6 @@ public function testSet()
         $this->assertEquals($foo, $sc->get('foo'), '->set() sets a service');
     }
 
-    public function testSetWithNullResetTheService()
```

Comment:
@stof i moved it back down, after other legacy tests. This is my first time on this, so i wasnt sure.